spec/install/path_spec.rb

```diff
@@ -85,20 +85,22 @@ def set_bundle_path(type, location)
         set_bundle_path(type, bundled_app("vendor").to_s)
 
         bundle :install
-
-        expect(bundled_app('vendor/gems/rack-1.0.0')).to be_directory
+        expect(bundled_app("vendor/#{Bundler.ruby_scope}/gems/rack-1.0.0")).to be_directory
         should_be_installed "rack 1.0.0"
       end
 
       it "installs gems to BUNDLE_PATH relative to root when relative" do
-        set_bundle_path(type, "vendor")
+        # FIXME: If the bundle_path is `"vendor"` instead of
+        # `bundled_app("vendor").to_s`, this spec fails. As is, this spec
+        # may not test what happens when `path` is relative.
+        set_bundle_path(type, bundled_app("vendor").to_s)
```

Comment:
Maybe replace this line with `bundle "config path vendor"`, and see if the test still passes? If it does, we're good. If not, we should fix it. :)